src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -1012,6 +1013,30 @@ export class SearchInterface extends RootComponent implements IComponentBindings
         }
       });
     });
+    if (this.FacetDuplicate.length) {
+      this.logger.warn(
+        `The following facets have duplicate id/field:`,
+        this.FacetDuplicate,
+        `Ensure that each facet in your search interface has a unique id.`
+      );
+    }
+  }
+
+  private get FacetDuplicate() {
```

Comment:
Name should not be capitalized, I would go with Sami's advice and call the method `duplicateFacets`

Reply:
Also duplicate can be multiple facets as it is an array so -> duplicates